Management/src/SecureEndpoints/SecureEndpointsWithBasicAuth/Controllers/HomeController.cs

```diff
@@ -46,20 +46,20 @@ public IActionResult About()
     public async Task<IActionResult> LogOff()
     {
         await HttpContext.SignOutAsync();
-        return RedirectToAction(nameof(HomeController.Index), "Home");
+        return RedirectToAction(nameof(Index), "Home");
     }
 
     [HttpGet]
     [Authorize]
     public IActionResult Login()
     {
-        return RedirectToAction(nameof(HomeController.Index), "Home");
+        return RedirectToAction(nameof(Index), "Home");
     }
 
     public IActionResult Manage()
     {
         ViewData["Message"] = "Manage accounts using UAA or CF command line.";
-        return View();
+        return View("PageWithMessage");
```

Comment:
I don't understand why this exists at all. There's no UI for login/logout/manage etc, so I suspect this was just copied from the UAA sample. Can it all be removed?

I was able to test this using the following .http file, which I generated with the help of https://www.debugbear.com/basic-auth-header-generator:

```http
GET http://localhost:5000/actuator
Authorization: Basic YWN0dWF0b3JVc2VyOmFjdHVhdG9yUGFzc3dvcmQ=
```

There are also UAA references in this project, including a "fortunes.read" scope, which I suspect is actually impossible to reach.